components/autocomplete/index.js

```diff
@@ -231,9 +232,15 @@ export class Autocomplete extends Component {
 		}
 	}
 
-	loadOptions( index ) {
-		this.props.completers[ index ].getOptions().then( ( options ) => {
-			const keyedOptions = map( options, ( option, i ) => ( { ...option, key: index + '-' + i } ) );
+	/**
+	 * Load options for an autocompleter.
+	 *
+	 * @param  {number} index    The autocompleter index.
+	 * @param  {string} query The query, if any.
+	 */
+	loadOptions( index, query ) {
+		this.props.completers[ index ].getOptions( query ).then( ( options ) => {
+			const keyedOptions = map( options, ( option, i ) => ( { ...option, key: index + '-' + ( option.value.slug ? option.value.slug : i ) } ) );
```

Comment:
This line is excessively long, and should be split across multiple lines to improve readability.

Reply:
addressed in https://github.com/WordPress/gutenberg/pull/4406/commits/887b091af2bc902d44d646a1b4359bf7835bb629